lib/gigya_flutter_plugin.dart

```diff
@@ -204,6 +204,15 @@ class GigyaSdk with DataMixin {
         return _timeoutError();
       });
 
+  /// Link social account to existing site account.
```

Comment:
Remove method.
Resolving should be applied via the resolving flow. Of course, you can add methods like this cause they all use the same channel invocation. But this is not what we intend to expose to the client.
In any case, the method name is "link to social" and the actual invocation is "link to site" which is incorrect.